.eslintrc.js

```diff
@@ -308,6 +309,10 @@ module.exports = {
         /**
          * react
          */
+        "react/no-string-refs": "off", // on in recommended, but #legacy-code
+        "react/no-find-dom-node": "off", // on in recommended, but #legacy-code
+        "react/display-name": "off", // on in recommended, but we have many components that don't have a displayName
+        "react/no-unescaped-entities": "off", // on in recommended, but #legacy-code
```

Comment:
I've turned off a few of the `react/recommended` rules because they were pretty pervasive. Some adventurous person can pick these off if that sounds like fun. :)